install.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-MBGLVERSION='3.4.1'
+MBGLVERSION='3.4.2'
```

Comment:
Since we’re using `circleStrokeWidth` and `circleStrokeColor`, this should point to a v3.5.0 beta, right?